backend/core/socketio_config.py

```diff
@@ -468,6 +497,12 @@ async def chat_message(sid, msg):
         session.close()
 
 
+@sio_server.on("read_message")
+async def read_message(sid, chat_session_id):
+    session = Session(engine)
+    handle_read_message(session=session, chat_session_id=chat_session_id)
```

Comment:
Is it possible to use background_tasks.add_task? and return immediately so no workload on the socket